scripts/base/frameworks/analyzer/dpd.zeek

```diff
@@ -32,13 +32,16 @@ export {
 	## Ignore violations which go this many bytes into the connection.
 	## Set to 0 to never ignore protocol violations.
 	option ignore_violations_after = 10 * 1024;
+
+	## Add removed services to conn.log, with a - in front of them.
+	option track_removed_services_in_connection = F;
```

Comment:
Naming and debatable, but wonder if `Conn::track_service_removals` is shorter and just as descriptive compared to `DPD::track_removed_services_in_connection`, even if it's in the other module (which still makes some sense though as that's where the log is created). Possibly also easier to find, given one would likely look into conn/main.zeek for logging of conn.log. A comment that the actual logic happens in `dpd.zeek` probably would help.

Reply:
I would like for this to remain in this module, so that dpd.zeek does not have to include conn.zeek.

I am happy to add a comment to conn/main.zeek. Is that good enough for you?

Reply:
Including `conn/main.zeek` isn't great. You could open the `Conn` module within `dpd.zeek` just to introduce the option.

I find the current name awkward and better fitting in `Conn` given the potential of this being a non-niche/interesting/popular option (I agree with you that default `T` could be considered) . But, I'm easily stubborn, so leaving it up to you or someone else chiming in - resolving :-)

Reply:
Using the conn namespace in this is a fun little hack, but I am not entirely sure about that :)

If this option actually becomes popular, we could just flup the default, and no one ever needs to use it.